charts/dex/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if and .Values.servicemonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "dex.fullname" . }}
+{{- if .Values.servicemonitor.namespace }}
+  namespace: {{ .Values.servicemonitor.namespace }}
+{{- else }}
+  namespace: {{ .Release.Namespace | quote }}
+{{- end }}
+  labels:
+    prometheus: {{ .Values.servicemonitor.prometheusInstance }}
+    app: {{ template "dex.name" . }}
+{{- include "dex.labels" . | indent 4 }}
+{{- if .Values.servicemonitor.labels }}
+{{ toYaml .Values.servicemonitor.labels | indent 4}}
+{{- end }}
+spec:
+  jobLabel: {{ template "dex.fullname" . }}
+  selector:
+    matchLabels:
+      {{- include "dex.selectorLabels" . | indent 6 }}
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
+  endpoints:
+  - port: telemetry
+    path: "/metrics"
```

Comment:
Isn't /metrics the default?

Reply:
Right. Could probably be omitted.

Reply:
No, no default that I am aware of. Docs: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#endpoint